package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "multiserver",
   "description": "",
-  "version": "1.2.3",
+  "version": "1.2.4",
```

Comment:
no biggie, but please leave changing the version number of the package to the maintainer